apps/website/src/index.css

```
@@ -0,0 +1,82 @@
+@tailwind base;
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)